apps/events/migrations/0002_remove_reservation_email_event_created_by_and_more.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 5.0.1 on 2024-02-23 23:01
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('events', '0001_initial'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='reservation',
+            name='email',
+        ),
+        migrations.AddField(
+            model_name='event',
+            name='created_by',
+            field=models.ForeignKey(default=None, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='reservation',
+            name='user',
+            field=models.ForeignKey(default=None, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='speaker',
+            name='last_updated_by',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL),
+        ),
+        migrations.AlterField(
+            model_name='basemodel',
+            name='id',
+            field=models.UUIDField(default='a4e542df-112b-4c35-982c-14da76a0d88b', editable=False, help_text='Unique identifier for this object.', primary_key=True, serialize=False, verbose_name='id'),
+        ),
+        migrations.AlterField(
+            model_name='eventtag',
+            name='id',
+            field=models.UUIDField(default='f7035ad1-6b09-4601-a040-c2766407ca68', editable=False, help_text='Unique identifier for this object.', primary_key=True, serialize=False, verbose_name='id'),
+        ),
+        migrations.AlterField(
+            model_name='speaker',
+            name='id',
+            field=models.UUIDField(default='0058691f-d433-4f1a-9697-379ad2e075c7', editable=False, help_text='Unique identifier for this object.', primary_key=True, serialize=False, verbose_name='id'),
+        ),
+    ]
```

Comment:
Can we have just a single migration file please. so you can go ahead and delete all the other migration files

Reply:
Okay